HPC/Gromacs.md

```diff
@@ -0,0 +1,122 @@
+Gromacs is one of the most widely used molecular dynamics software packages. Below is the instructions to build and run Gromacs on Hpc7g instances.
+
+## Build Gromacs
+* build cmake
+```
+wget https://github.com/Kitware/CMake/releases/download/v3.26.0-rc4/cmake-3.26.0-rc4.tar.gz
+tar xf cmake-3.26.0-rc4.tar.gz 
+cd cmake-3.26.0-rc4
+./bootstrap
+sudo make install
+```
+* build binutils
```

Comment:
Ubuntu-22.04 comes with binutils-2.38.

Reply:
Some EC2 instances have older version of binutils and need the newer version in order to generate SVE/SIMD flags based on the architecture.

Reply:
I can remove this part.  Below is the list of images supported in the latest parallel cluster (v3.5.0).  Maybe we can ask ubuntu 22.04 or al2023 to be added. 
```
pcluster list-official-images
{
  "images": [
    {
      "amiId": "ami-0542b79ed51567d9f",
      "os": "ubuntu1804",
      "name": "aws-parallelcluster-3.5.0-ubuntu-1804-lts-hvm-x86_64-202302151336 2023-02-15T13-40-36.600Z",
      "version": "3.5.0",
      "architecture": "x86_64"
    },
    {
      "amiId": "ami-017c06b6551a8ae93",
      "os": "centos7",
      "name": "aws-parallelcluster-3.5.0-centos7-hvm-x86_64-202302151337 2023-02-15T13-40-33.680Z",
      "version": "3.5.0",
      "architecture": "x86_64"
    },
    {
      "amiId": "ami-0887887ce422f646e",
      "os": "alinux2",
      "name": "aws-parallelcluster-3.5.0-amzn2-hvm-x86_64-202302151337 2023-02-15T13-40-45.464Z",
      "version": "3.5.0",
      "architecture": "x86_64"
    },
    {
      "amiId": "ami-0fbd23ce8cbb7250f",
      "os": "ubuntu1804",
      "name": "aws-parallelcluster-3.5.0-ubuntu-1804-lts-hvm-arm64-202302151336 2023-02-15T13-41-21.284Z",
      "version": "3.5.0",
      "architecture": "arm64"
    },
    {
      "amiId": "ami-0e455a4e30f312629",
      "os": "ubuntu2004",
      "name": "aws-parallelcluster-3.5.0-ubuntu-2004-lts-hvm-x86_64-202302151337 2023-02-15T13-40-53.399Z",
      "version": "3.5.0",
      "architecture": "x86_64"
    },
    {
      "amiId": "ami-094e11f92e31a8a49",
      "os": "alinux2",
      "name": "aws-parallelcluster-3.5.0-amzn2-hvm-arm64-202302151337 2023-02-15T13-41-09.848Z",
      "version": "3.5.0",
      "architecture": "arm64"
    },
    {
      "amiId": "ami-011e654a7de2d8097",
      "os": "ubuntu2004",
      "name": "aws-parallelcluster-3.5.0-ubuntu-2004-lts-hvm-arm64-202302151337 2023-02-15T13-41-26.602Z",
      "version": "3.5.0",
      "architecture": "arm64"
    }
  ]
}
```